diff --git a/kritatwitch/vendor/urllib3/contrib/pyopenssl.py b/kritatwitch/vendor/urllib3/contrib/pyopenssl.py
index 50a07d5..ac922ad 100644
--- a/kritatwitch/vendor/urllib3/contrib/pyopenssl.py
+++ b/kritatwitch/vendor/urllib3/contrib/pyopenssl.py
@@ -28,8 +28,8 @@ like this:
 .. code-block:: python
 
     try:
-        import urllib3.contrib.pyopenssl
-        urllib3.contrib.pyopenssl.inject_into_urllib3()
+        import urllib3.contrib.pyopenssl as pyopenssl
+        pyopenssl.inject_into_urllib3()
     except ImportError:
         pass
 
diff --git a/kritatwitch/vendor/urllib3/contrib/securetransport.py b/kritatwitch/vendor/urllib3/contrib/securetransport.py
index 6c46a3b..abe03d2 100644
--- a/kritatwitch/vendor/urllib3/contrib/securetransport.py
+++ b/kritatwitch/vendor/urllib3/contrib/securetransport.py
@@ -19,8 +19,8 @@ contrib module. So...here we are.
 
 To use this module, simply import and inject it::
 
-    import urllib3.contrib.securetransport
-    urllib3.contrib.securetransport.inject_into_urllib3()
+    import urllib3.contrib.securetransport as securetransport
+    securetransport.inject_into_urllib3()
 
 Happy TLSing!
 
